iohk-monitoring/src/Cardano/BM/Output/Monitoring.lhs

```diff
@@ -208,4 +214,13 @@ evalMonitoringAction mmap logObj@(LogObject logname _ _) =
             : acc
     -- catch all
     updateEnv env _ = env
+
+    performAction (CreateMessage sev alertMessage) = do
+        lometa <- mkLOMeta sev Public
+        Trace.traceNamedObject sbtrace (lometa, MonitoringEffect (MonitorAlert alertMessage))
+    performAction (SetGlobalMinimalSeverity sev) =
+        setMinSeverity config sev
+    performAction (AlterSeverity loggerName sev) =
+        setSeverity config loggerName (Just sev)
+
```

Comment:
MonitoringEffect is sent to the switchboard, the other actions are executed right here. I thought we had discussed that MonitoringEffect will have more actions (constructors) and all be sent to the switchboard which then evaluates them (with side-effects). That way we can also treat them with the existing machinery..

 \+ add a name context to the sbtrace.